ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -715,6 +716,34 @@ public void testErrorPrecedenceWithSpecialCase() throws Exception {
     }
   }
 
+  /**
+   * Test the case where originating dc returns 2 Disk_Unavailable and 1 Not_Found and rest replicas return Not_Found.
```

Comment:
what if 2 Disk_Unavailable and 1 Deleted?

Reply:
As long as `(# of Disk_Unavailable + # of Not_Found) > (total replica count - success target)`, we consider it Not_Found